src/components/Fire.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import { getFirebaseAppRemoteConfig } from '../firebase';
+const Fire = () => {
```

Comment:
use Hooks for this
create new file in hooks folder called
const useFirebaseConfig = () => {}